doc/src/manual/getting-started.md

```diff
@@ -136,6 +136,10 @@ julia [switches] -- [programfile] [args...]
     In Julia 1.0, the default `--project=@.` option did not search up from the root
     directory of a Git repository for the `Project.toml` file. From Julia 1.1 forward, it
     does.
+ 
+## Startup Configuration
+
+  A startup file may be placed in `~/.julia/config/startup.jl` containing Julia commands to be executed each time Julia starts up.  Many people like placing frequently used packages in a startup `using` command such as `using LinearAlgebra, Statistics`.
```

Comment:
This is already explained above in the same page: https://github.com/JuliaLang/julia/pull/36619/files#diff-81c32d0e89a1c0a596204cfcb0398d25R78-R79 I feel it less needed to explain it again.